### PR TITLE
test: [M3-8105] - Clean up cy.intercept calls in resize-linode

### PR DESCRIPTION
**Pull request by AzureLatte**
## Description 📝
Clean up cy.intercept calls in resize-linode

## Changes  🔄
List any change relevant to the reviewer.

- Replace calls to cy.intercept with intercept utils in linodes.ts.
- Adds interceptLinodeResize() util

## How to test 🧪
We can rely on CI to confirm that this test has not been broken by these changes, but to run the test locally you can use this command:
```
yarn cy:run -s "cypress/e2e/core/linodes/resize-linode.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [x] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by bnussman-akamai — APPROVED** [review]
🧼

**Review by jaalah-akamai — APPROVED** [review]

File: packages/manager/cypress/e2e/core/linodes/resize-linode.spec.ts
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/linodes/resize-linode.spec.ts:1]: Seeing one call to `cy.intercept()` that slipped through the cracks!

`ln 40`
```typescript
      cy.intercept(
        'POST',
        apiMatcher(`linode/instances/${linode.id}/resize`)
      ).as('linodeResize');
```

(GitHub wouldn't let me comment on those specific lines for some reason 😅)

File: packages/manager/cypress/e2e/core/linodes/resize-linode.spec.ts [hunk 1]
```diff
@@ -80,10 +78,7 @@ describe('resize linode', () => {
 
       containsVisible('OFFLINE');
 
-      cy.intercept(
-        'POST',
-        apiMatcher(`linode/instances/${linode.id}/resize`)
-      ).as('linodeResize');
+      mockGetFeatureFlagClientstream().as('linodeResize');
```
Comment (jdamore-linode) [inline packages/manager/cypress/e2e/core/linodes/resize-linode.spec.ts:81]:
```suggestion
      interceptLinodeResize().as('linodeResize');
```
(This also applies to the replacements later in the file)

Interestingly the tests still run and pass just fine -- it calls into question whether the intercepts are even really necessary in these tests!

**Review by jdamore-linode — COMMENTED** [review]
Couple small things, but I think it raises a more interesting question: if the test seems to run just fine without the intercepts for the resize requests, would it be better to just get rid of them altogether? Any thoughts?

File: packages/manager/cypress/e2e/core/linodes/resize-linode.spec.ts (hunk 1 above)
Comment (jdamore-linode) [reply packages/manager/cypress/e2e/core/linodes/resize-linode.spec.ts]: Haha we've all been there 😅

**Review by jdamore-linode — APPROVED** [review]
Great work @AzureLatte!

**Pull request merged**